<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProjectSettingsPage</class>
 <widget class="QDialog" name="ProjectSettingsPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>550</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>450</width>
    <height>550</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="Title">
     <property name="font">
      <font>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>Settings</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>8</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QLabel" name="ProjectName">
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Project Name</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="ProjectNameEdit">
     <property name="styleSheet">
      <string notr="true">border-radius:4px;
padding:4px;
</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label">
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Project Users</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="UsersListsLayout">
     <item>
      <layout class="QVBoxLayout" name="CurrentProjectUsrsLayout">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Current project users</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListWidget" name="CurrentUsers">
         <property name="styleSheet">
          <string notr="true">border-radius:4px;
padding:4px;</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="AvailableUsrsLayout">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Available users</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListWidget" name="AvailableUsers">
         <property name="styleSheet">
          <string notr="true">border-radius:4px;
padding:4px;</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLineEdit" name="SearchUsers">
     <property name="styleSheet">
      <string notr="true">border-radius:4px;
padding:4px;</string>
     </property>
     <property name="placeholderText">
      <string>Search Users</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="SaveBtn">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(5, 109, 120);
color: rgb(255, 255, 255);
border-radius:8px;
padding-top:4px;
padding-bottom:4px;
padding-left:25px;
padding-right:25px;</string>
     </property>
     <property name="text">
      <string>Save</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
